testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/MultipleTabsLoginTest.java

```diff
@@ -246,22 +246,25 @@ private void multipleTabsParallelLogin(BrowserTabUtil tabUtil) {
         oauth.openLoginForm();
         loginPage.assertCurrent();
         getLogger().info("URL in tab1: " + driver.getCurrentUrl());
-
         // Open new tab 2
         tabUtil.newTab(oauth.getLoginFormUrl());
         assertThat(tabUtil.getCountOfTabs(), Matchers.equalTo(2));
         loginPage.assertCurrent();
         getLogger().info("URL in tab2: " + driver.getCurrentUrl());
-
         // Wait until authentication session expires
         setTimeOffset(7200000);
 
+        //triggers the postponed function in authChecker.js to check if the auth session cookie has changed
+        try {
+            Thread.sleep(2000);
```

Comment:
Can this be replaced with `WaitUtils.sleep(2000)` ? Even much more ideal would be to replace it with something like: `wait.until(checkSomehowIfCookieChanged())`, which is not blocking operation and at the same time, it is more reliable. See various other operations in the `WaitUtils` class for the inspiration. But at the same time, I am not 100% sure if this one can be reliably used... Can you doublecheck? Same applies for other similar operations in this test.

Reply:
I had already tried, but unfortunately there are no changes to intercept. The sleep is only to get past the timeout in `authChecker.js`, otherwise the test would be failed because with the `setTimeOffset` the time has moved forward, but not for the browser.

Reply:
Ok, I am resolving this discussion, Thanks!